list.html

```diff
@@ -0,0 +1,81 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet" type="text/css" href="application.css">
+</head>
+
+<body class="body">
+  <div class="books-container row">
+    <div class="book column shadow-container">
+      <div class="book-info">
+        <img src="./assets/book-cover.png" alt="Book cover" class="book-cover-list">
```

Comment:
Why not writing this instead

```suggestion
        <img src="./assets/book-cover.png" alt="Book cover" class="book-info-cover">
```

Since, it will be like working with that component `book-info-`